### PR TITLE
adding `get_available_metrics` method

### PR DESCRIPTION
**Pull request by immortal3**
Based on databaset column, it tries to return all available metrics which are suitable for evalution

### CODE REVIEW

File: src/ragas/metrics/__init__.py [hunk 1]
```diff
@@ -15,6 +17,37 @@
...
+    context_precision,
+    context_utilization,
+    context_recall,
+    context_relevancy,
+    answer_relevancy,
+    context_entity_recall,
+]
+
+
+def get_available_metrics(ds: Dataset) -> list[Metric]:
```
Comment (jjmachan) [inline src/ragas/metrics/__init__.py:36]: could you move this to a new file called `metrics/utils` and import it here?
that will be much cleaner

**Review by jjmachan — CHANGES_REQUESTED** [review]
thanks again for the PR, just one tiny change and we should be good to go

File: src/ragas/metrics/__init__.py (hunk 1 above)
Comment (immortal3) [reply src/ragas/metrics/__init__.py]: yeah, my plan was to do in utils.py only, but it was having circular dependency. Raised new changes which brings this function to `metrics/utils`

**Review by jjmachan — APPROVED** [review]
thanks a lot!

**Pull request merged**